source/utilities.tex

```diff
@@ -17786,7 +17786,7 @@
 \indexlibrary{\idxcode{remove_cv}}%
 \tcode{template<class T>\br
  struct remove_cv;}                 &
- The member typedef \tcode{type} shall be the same as \tcode{T}
+ The member typedef \tcode{type} is the same as \tcode{T}
```

Comment:
Maybe "`type` denotes the type formed by removing any top-level cv-qualifiers from `T`"